browser/src/UI/components/Notifications.less

```diff
@@ -0,0 +1,91 @@
+@import (reference) "./common.less";
+
+@icon-size: 30px;
+@font-family-monospace: Consolas, "Liberation Mono", Menlo, Courier, monospace;
```

Comment:
This might be a good candidate to have in `common.less` - same for a few other constants.